src/pages/landingPage/index.tsx

```diff
@@ -1,13 +1,87 @@
-import { Container } from '@mui/material'
+import { Container, Grid } from '@mui/material'
+import { hero, its, discover, join, appointments } from './styles'
+import { Link } from 'react-router-dom'
 
 const LandingPage = ()=> {
   
 
   return (
-    <Container>
-     
-      LandingPage
-    </Container>
+    <>
+      <Container sx={hero}> 
+      <h1 className='way'>WAY </h1>
```

Comment:
Use MUI Typography instead of p tags and h1

Reply:
done